### ISSUE TITLE
Add Flag `--image-patterns` to cmd

### ISSUE THREAD

**Issue by panpan0000** [bitnami-labs/charts-syncer#165]
Currently , if we would like to enable `relocateContainerImages=true` feature to push image to target and modify the image repo in new helm chart,  it requires the source helm chart has embeded `.relok8s-images.yaml`.

But unfortunately,  Not all  helm chart has this file, even the bitnami helm charts. Maybe only those charts in `https://charts.app-catalog.vmware.com` will include this file in the world.

The `relok8s` (https://github.com/vmware-tanzu/asset-relocation-tool-for-kubernetes) has a flag `--image-patterns` to specific an external `.relok8s-images.yaml` , which it's more friendly , not making it a hard requirement of source chart.
 
So design proposal: add `--image-patterns` :

People can create an external pattern file `my-pattern.yaml` (external means standalone, out of helm charts)

Usage 
```
charts-syncer sync --image-patterns  my-pattern.yaml
```

In this way, every charts can be "relocated", not limited to those in catalog.vmware.com.

**Comment by panpan0000** [comment]
Hope this is can fulfilled and make this open source tool more useful and friendly.

**Comment by HammerNL89** [comment]
Yes this would be a nice feature, currently I'm forced to use a separate tool for syncing the required images which adds complexity.
